march_rqt_input_device/src/march_rqt_input_device/input_device_view.py

```diff
@@ -148,6 +148,14 @@ def __init__(self, context):
             self.create_button('gait_tilted_path_second_start',
                                callback=lambda: self.controller.publish_gait('gait_tilted_path_second_start'))
 
+        gait_tilted_path_first_end = \
+            self.create_button('gait_tilted_path_first_end',
+                               callback=lambda: self.controller.publish_gait('gait_tilted_path_first_end'))
+
+        gait_tilted_path_second_end = \
+            self.create_button('gait_tilted_path_second_end',
+                               callback=lambda: self.controller.publish_gait('gait_tilted_path_second_end'))
+
```

Comment:
These names are not the same as in project-march/march#441

Reply:
The starting steps were already in the IPD, so only the two ending steps were added.